atom/browser/web_contents_preferences.cc

```diff
@@ -160,7 +160,14 @@ void WebContentsPreferences::AppendExtraCommandLineSwitches(
   if (window) {
     bool visible = window->IsVisible() && !window->IsMinimized();
     if (!visible)  // Default state is visible.
-      command_line->AppendSwitch("hidden-page");
+      command_line->AppendSwitch(switches::kHiddenPage);
+  } else {
```

Comment:
A WebContents without window doesn't not necessarily mean it is a webview, it can be an anonymous WebContents created by `electron.webContents.create()`, this API is not public yet but we might make it public in future.

The safe way is to check whether `guest_instance_id` is set.